### PR TITLE
Upgrade SvelteKit to 1.0.0 stable release

### LINKED ISSUE #56: Upgrade to SvelteKit stable release
# Summary

## Problem

SvelteKit has released a 1.0.0, and we aren't using it.

### Impact

No immediate impact, but moving to stable brings us forward past breaking changes.

## Solution

Upgrade to latest SvelteKit stable release.

### PR DESCRIPTION
**Pull request by bgins**
# Description

This PR implements the following features:

- [x] Upgrade `@sveltejs/kit` to `1.0.1`
- [x] Upgrade `@sveltejs/adapter-static` to `1.0.0`
- [x] Update npm scripts to use `vite` and add `vite` config
- [x] Update head and body template strings in `app.html` 
- [x] Set adapter static fallback route
- [x] Update rendering configs
- [x] Move global CSS file to `src` directory
- [x] Extend SvelteKit tsconfig and update path aliases
- [x] Add `trailingSlashAlways` config (works better with IPFS HTTP Gateway if more pages are added)
- [x] Add `rel="noreferrer"` to outbound links

## Link to issue

Closes #56 

## Type of change

- [x] Refactor (non-breaking change that updates existing functionality)

## Test plan (required)

The app should continue to work in local development, built, and published to Fission static site hosting.

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      <Column>
+        <slot />
+      </Column>
+    </Row>
+  </Grid>
+</Content>
+<Footer />
+
+<style>
+  /* Styles to normalize how the markdown-sourced content looks */
```
Comment (avivash) [inline src/routes/+layout.svelte:25]: should these be added to the `global.css` file instead?

**Review by avivash — APPROVED** [review]
One comment, but looks good otherwise! I've tested locally, but it may be good to test the prod build against Safari too to ensure we don't run into the same issue from the other day

File: src/routes/+layout.svelte (hunk 1 above)
Comment (bgins) [reply src/routes/+layout.svelte:25]: Yep, that would be a better place. Unfortunately, `global.css` is currently a blob of compiled CSS pulled directly from the [Carbon Components source code](https://github.com/carbon-design-system/carbon-components-svelte/blob/master/css/white.css), which will make it a bit awkward to move things there.

We're planning to refresh the design in this project. Let's look at moving the styles over at then.

**Comment by bgins** [comment]
> One comment, but looks good otherwise! I've tested locally, but it may be good to test the prod build against Safari too to ensure we don't run into the same issue from the other day

Tested on Safari. No issues, merging this in! 🚀

**Pull request merged**